radix-engine/src/blueprints/resource/non_fungible/non_fungible_resource_manager.rs

```diff
@@ -35,6 +35,8 @@ pub struct NonFungibleResourceManagerMutableFieldsSubstate {
 
 pub type NonFungibleResourceManagerTotalSupplySubstate = Decimal;
 
+pub const NON_FUNGIBLE_RESOURCE_MANAGER_DATA_STORE: CollectionIndex = 0u8;
```

Comment:
What's a collection index? Is this effectively the partition offset?

Or the partition offset - 1? (to save room for the tuple partition?)